create-configmap.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
This is going to drop a lot more files into the repos root directory. It might make sense to move the tasks into a subdir or maybe also the openshift or ansible content.

Reply:
should I move all of the playbooks/tasks files into `playbooks/`?

Reply:
> This is going to drop a lot more files into the repos root directory. It might make sense to move the tasks into a subdir or maybe also the openshift or ansible content.

There is already a lot of stuff in the root directory.  I suppose I could clean up this content - move it into sub-directories as part of this PR, but I think that would be better done in another separate PR.

Reply:
> should I move all of the playbooks/tasks files into `playbooks/`?

I am not sure how much the other files and the playbook relate, here are possible ideas:

/ansible/tasks/ - for tasks
/ansible/ - for playbooks
/openshift - for OpenShift helpers and files

an alternative could be:

/openshift - for OpenShift tools including the playbook
/openshift/files - for files used by tools and playbook
/openshift/task - for Ansible tasks

> > This is going to drop a lot more files into the repos root directory. It might make sense to move the tasks into a subdir or maybe also the openshift or ansible content.
> 
> There is already a lot of stuff in the root directory. I suppose I could clean up this content - move it into sub-directories as part of this PR, but I think that would be better done in another separate PR.

I guess this depends on how much the files/tools depend on each other. If it is not much extra work to do the extra PR, it makes reviewing things and moving forward easier.

Reply:
> > should I move all of the playbooks/tasks files into `playbooks/`?
> 
> I am not sure how much the other files and the playbook relate, here are possible ideas:
> 
> /ansible/tasks/ - for tasks
> /ansible/ - for playbooks

Done.

> /openshift - for OpenShift helpers and files

I don't want to move these because I don't want to disrupt the workflow of other users of this repo.  See the `README.md` for the directions for the old workflow that doesn't use ansible.

> 
> an alternative could be:
> 
> /openshift - for OpenShift tools including the playbook
> /openshift/files - for files used by tools and playbook
> /openshift/task - for Ansible tasks

See above.

> 
> > > This is going to drop a lot more files into the repos root directory. It might make sense to move the tasks into a subdir or maybe also the openshift or ansible content.
> > 
> > 
> > There is already a lot of stuff in the root directory. I suppose I could clean up this content - move it into sub-directories as part of this PR, but I think that would be better done in another separate PR.
> 
> I guess this depends on how much the files/tools depend on each other. If it is not much extra work to do the extra PR, it makes reviewing things and moving forward easier.

* I don't want to disrupt the existing workflow of other users of this repo
* I don't want to have too much unrelated stuff in this PR